controllers/delete-comment.controller.js

```diff
@@ -0,0 +1,14 @@
+const { deleteCommentById } = require("../models/models")
+
+const deleteComment = (req, res, next) => {
+    const {comment_id} = req.params
+    deleteCommentById(comment_id)
+    .then((result) => {
+        res.status(204).send(result)
```

Comment:
There is no result to respond with here
You either want `res.sendStatus(204)` or `res.status(204).send()`